cql3/query_processor.hh

```diff
@@ -462,6 +462,8 @@ public:
 
     void reset_cache();
 
+    bool topology_global_queue_empty();
```

Comment:
nit: Can be marked const

Reply:
Tried it, but compiler said he disagrees, because methods called from within are not marked const.

Reply:
One of the methods is storage_service::topology_global_queue_empty() introduced in this set, which can also be marked const from the very beginning. Another method must be remote() which tries to grab gate holder, but this grabbing seems a waste of cycles and is confusing, because you're not going to hold the remote of q.p. for too long.

Reply:
> Tried it, but compiler said he disagrees, because methods called from within are not marked const.

Currently it just calls:
```
    return !_topology_state_machine._topology.global_request.has_value();
```

How is it not const?

Reply:
So actually there are 2 methods named the same way. I decorated one of it with `const` specifier, the other one I couldn't, as it was calling non-const methods.